pkg/pillar/types/zedmanagertypes.go

```diff
@@ -247,6 +247,10 @@ type SnapshottingStatus struct {
 	HasRollbackRequest bool
 	// ActiveSnapshot contains the id of the snapshot to be used for the rollback.
 	ActiveSnapshot string
+	// RollbackInProgress indicates whether a rollback is in progress for the app instance.
```

Comment:
When does the "inprogress" end?
Is this set while we are doing the rollback operation and cleared when the app instance is back running based on the snapshotted config and volumes? Or does it remain set as long as the app instance is running from an old snapshot?
In the latter case a different name would make sense e.g., "InRollback" or "NotUsingLatestConfig" or "UsingSnapshottedConfig".